tests/controllers/UsersControllerTest.php

```diff
@@ -0,0 +1,166 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: liam
+ * Date: 10/09/16
+ * Time: 12:54
+ */
+
+namespace JoindinTest\Controller;
+
+
+class UsersControllerTest extends \PHPUnit_Framework_TestCase
+{
+
+    /**
+     * Ensures that if the deleteUser method is called and no user_id is set,
+     * an exception is thrown
+     *
+     * @return void
+     *
+     * @test
+     * @expectedException \Exception
+     * @expectedExceptionMessage You must be logged in to delete data
+     */
+    public function testDeleteUserWithNoUserIdThrowsException(){
```

Comment:
From what I've seen so far the Codestyling here is different from the styling used in the ```src```-folder. Is that on purpose? The ```tests``` -folder is currently not checked by phpcs as I've noticed.